resfit/fit_resonator/fit_S_data.py

```diff
@@ -777,7 +777,44 @@ def monte_carlo_fit(xdata= None,ydata=None,parameter=None,Method = None):
         stop_MC = True
     return parameter,stop_MC, error
 
-
+def get_header(line: str):
```

Comment:
should specify a return type here as well 
```
def get_header(line: str) -> Tuple[str, str]:
```
You'll have to import Tuple at the top like so: 
`from typing import Tuple`